src/app/features/tech-record-summary/tech-record-summary.component.spec.ts

```diff
@@ -0,0 +1,81 @@
+import { TechRecordSummaryComponent } from './tech-record-summary.component';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { TechRecordModel, VehicleTypes, StatusCodes, FuelTypes, VehicleConfigurations, EuVehicleCategories, VehicleSizes } from '@models/vehicle-tech-record.model';
+import { By } from '@angular/platform-browser';
+
+
+describe('TechRecordSummaryComponent', () => {
+    let component: TechRecordSummaryComponent;
+    let fixture: ComponentFixture<TechRecordSummaryComponent>;
+
+    const fakeTechRecord: TechRecordModel = {
```

Comment:
Create functions using `createMock` like in `src/mocks/mock-test-result.ts`. Saves time and keeps test readable.